tests/unit/aiplatform/test_pipeline_jobs.py

```diff
@@ -850,6 +854,86 @@ def test_run_call_pipeline_service_create_with_failure_policy(
             gca_pipeline_state.PipelineState.PIPELINE_STATE_SUCCEEDED
         )
 
```

Comment:
Could you please also update the `test_clone_pipeline_job_with_all_args` with `input_artifacts`? Thanks!

Reply:
Done.